src/operator/convolution.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+import edgeHandlingMethod from './edgeHandlingMethod'
+
+//convolution using a square kernel
+export default function convolution(newImage, kernel, edgeHandling){
```

Comment:
edgeHandling is obviously an option. Options should be all put in an object as last parameter of the function (see [grey transform](https://github.com/image-js/core/blob/master/src/transform/grey/grey.js#L10)). As you can see, default values for options can be specified at this level.